### PR TITLE
Ansible updates

### PR DESCRIPTION
**Pull request by eparis**
Just minimal etcd and kube. But use tags, rename minion->node. Tiny readme update.

### CODE REVIEW

**Comment by eparis** [comment]
@nzwulfin

**Comment by bgrant0607** [comment]
Failed boilerplate checks.

**Comment by eparis** [comment]
hopefully that fixed the boilerplate.

File: contrib/ansible/README.md [hunk 1]
```diff
@@ -4,13 +4,15 @@ This playbook helps you to set up a Kubernetes cluster on machines where you
...
-* Record the IP address of the machine you want to be your etcd server (often same as master)
-* Record the IP addresses of the machines you want to be your minions. (master can be a minion)
+* Record the IP address/hostname of which machine you want to be your master (only support a single master)
+* Record the IP address/hostname of the machine you want to be your etcd server (often same as master, only one)
+* Record the IP addresses/hostname of the machines you want to be your nodes. (the master can also be a node)
 
-Stick the system information into the 'inventory' file.
+### Configure the inventory file
+
+Stick the system information gathered above into the 'inventory' file.
```
Comment (jeffbean) [inline contrib/ansible/README.md]: I would also show you can create more 'inventory' files and show how to run that with`setup.sh`

I assume it is:

    $ INVENTORY=other_inventory_file ./setup.sh

**Comment by bgrant0607** [comment]
@eparis Could you please suggest a reviewer?

**Comment by eparis** [comment]
@bgrant0607 I don't know. both @nzwulfin and @jeffbean are good with ansible. Not sure who to call the 'reviewer' though.  I'm 'out' this week, but might be able to find a Red Hat OpenShift person next week to look at well...

File: contrib/ansible/roles/master/tasks/main.yml [hunk 2]
```diff
@@ -1,4 +1,13 @@
 ---
+- name: Install kubernetes
+  action: "{{ ansible_pkg_mgr }}"
+  args:
+        name: kubernetes-master
```
Comment (jeffbean) [inline contrib/ansible/roles/master/tasks/main.yml]: Right now `kubernetes-master` and `kubernetes-node` packages are only in the testing repo for CentOS7.

We should just call that out in the README, if the assumption is the repositories for whatever distro you are using are already setup.

```
[root@kube-minion-test-01 ~]# yum --showduplicates list kubernetes* | expand
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: cosmos.cites.illinois.edu
 * extras: centos.mia.host-engine.com
 * updates: centos.mbni.med.umich.edu
Installed Packages
kubernetes.x86_64               0.18.0-0.2.git5520386.el7         @virt7-testing
kubernetes-master.x86_64        0.18.0-0.2.git5520386.el7         @virt7-testing
kubernetes-node.x86_64          0.18.0-0.2.git5520386.el7         @virt7-testing
Available Packages
kubernetes.x86_64               0.9.0-0.3.git96af0c3.el7          extras        
kubernetes.x86_64               0.15.0-0.3.git0ea87e4.el7         extras 
```

**Comment by eparis** [comment]
closing, this has been subsumed by other PRs

**Pull request closed without merging**